CenterForOpenScience/osf.io
# OSF registries landing page [OSF-7222]

## Purpose

Need a landing page for registries.

## Changes

Add a landing page for registries.

## Side effects

None.


## Ticket
OSF-7222
https://openscience.atlassian.net/browse/OSF-7222


## QA Notes
1. Make sure all links lead to where you expect. 
2. Make sure no text on the page refers to preprints still, I think I caught it all.
3. Make sure all image render properly
4. Also check on small windows to make sure it doesn't look terrible/broken.
